docs/modules/configuration/pages/packagers/sdkman.adoc

```diff
@@ -48,7 +48,7 @@ include::partial$packagers/yaml/common-head.adoc[]
     command: MAJOR
 
     # The release notes URL to use.
-    # Defaults to `${release.<git>.releaseNotesUrl}.
+    # Defaults to `${release.<git>.releaseNotesUrl}`.
```

Comment:
Right now `<git>` doesn't appear in the doc. I hope this will be enough to fix it but it should be checked.